src/librustdoc/html/render.rs

```diff
@@ -1369,7 +1369,9 @@ impl Context {
             self.render_redirect_pages = self.maybe_ignore_item(&item);
         }
 
-        if item.is_mod() {
+        if item.attrs.list("doc").has_word("hidden") {
+            Ok(())
+        } else if item.is_mod() {
```

Comment:
It seems that the strip_hidden pass is supposed to be responsible for removing hidden items. Why is that pass not sufficient? Shouldn't the hidden items have been stripped by this point and not be seen here?